addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/zone.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+     <thing-type id="zone">
+
+	    <supported-bridge-type-refs>
+            <bridge-type-ref id="envisalinkbridge" />
+            <bridge-type-ref id="it100bridge" />
+        </supported-bridge-type-refs>
+
+        <label>DSC Alarm Zone</label>
+        <description>Represents a physical device such as a door, window, or motion sensor.</description>
+
+        <channels>
+            <channel id="zone_status" typeId="zone_status"/>
+            <channel id="zone_message" typeId="zone_message"/>
+            <channel id="zone_bypass_mode" typeId="zone_bypass_mode"/>
+            <channel id="zone_in_alarm" typeId="zone_in_alarm"/>
+            <channel id="zone_tamper" typeId="zone_tamper"/>
+            <channel id="zone_fault" typeId="zone_fault"/>
+            <channel id="zone_tripped" typeId="zone_tripped"/>
+        </channels>
+
+		<config-description>
+
+			<parameter name="partitionNumber" type="integer" min="1" max="8">
+				<label>Partition Number</label>
+				<description>The Partition Number (1-8).</description>
+				<default>1</default>
+			</parameter>
+
+			<parameter name="zoneNumber" type="integer" required="true" min="1" max="64">
+				<label>Zone Number</label>
+				<description>The Zone Number (1-64).</description>
+			</parameter>
+
+		</config-description>
+ 
+    </thing-type>
+
+    <channel-type id="zone_status">
+        <item-type>Contact</item-type>
```

Comment:
On the DSC Alarm a zone's status is either open or closed regardless of the device (contact, motion sensor, smoke detector, etc.) connected.  So, if a motion detector, for example, is triggered the zone is going to be shown as open.